lib/ifupdown.sh.in

```diff
@@ -1,11 +1,15 @@
 #!/bin/sh
 
+# This script should be symlinked into /etc/network/ip-pre-up.d/ and
```

Comment:
I'm doing this manually.  There are some automatic formatting tools out there, but I'm not using one.